components/Mission/Item.vue

```diff
@@ -27,7 +22,18 @@
             :mission-id="mission.id"
             class="mission-icon"
           />
-          <h1 class="title-label"><span>{{ title }}</span></h1>
+
+          <mission-countdown-timer
+            v-if="isCooldown"
+            :date="countdownTime"
+            :layout="layout"
+            :isFull="layout === 'small'"
```

Comment:
if `isFull` depends on `layout`, then no need to define props, use internal computed props instead